cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -275,6 +275,10 @@
     /var/lib/snapd/hostfs/var/lib/lxd r,
 
     # support for the mount namespace sharing
+    capability sys_ptrace,
+    # allow snap-confine to read /proc/1/ns/mnt
+    ptrace peer=unconfined,
```

Comment:
I ran a quick test without `capability sys_ptrace that showed green on the regression test. I chose to remove this extra permission. I think it was added when we tested various kernel fixes but I don't think it is required in practice now.